integration/images.go

```diff
@@ -151,19 +151,16 @@ func FindDockerFiles(dockerfilesPath string) ([]string, error) {
 // DockerFileBuilder knows how to build docker files using both Kaniko and Docker and
 // keeps track of which files have been built.
 type DockerFileBuilder struct {
-	// Holds all available docker files and whether or not they've been built
-	FilesBuilt           map[string]bool
+	// Holds all available docker files and whether or not they've been built*
```

Comment:
nite: remove star unless there is a reason to add it.

Reply:
done